addon/components/frost-select.js

```diff
@@ -322,6 +322,17 @@ export default Component.extend({
     this.$().focus()
   },
 
+  triggerFilterInput (filter) {
+    const inputTask = this.get('inputTask')
+    const onInput = this.get('onInput')
+
```

Comment:
This can be simplified with `const {inputTask, onInput} = this.getProperties('inputTask', 'onInput')`.

Reply:
I will prefer to keep as is for now to avoid modifying the behavior too much. If I do that and that one of the value is undefined the deconstruction will throw an exception.